lib/shelley/cardano-wallet.cabal

```diff
@@ -37,9 +37,9 @@ library
     , bech32
     , bech32-th
     , binary
-    , blockfrost-api >= 0.4 && < 0.5
-    , blockfrost-client >= 0.4 && < 0.5
-    , blockfrost-client-core >= 0.4 && < 0.5
```

Comment:
I think we can remove explicit version boundaries (this isn't a hard requirement for PR approval)